proposals/evaluation/uc-3-inheritance.md

```diff
@@ -79,6 +88,8 @@ A unauthenticated agent making a request to  `</weekly-status/2021-04-28/report.
 ```
 
```

Comment:
Is this applyMembers correct for an ACP on a `ldp:Resource`? I would have thought that only made sense on an `ldp:Container`

Reply:
You're correct, line 87 needs to go (and fullstop at the end of line 86).

Reply:
Thanks :)